tidb-lightning/tidb-lightning-overview.md

```diff
@@ -29,11 +23,11 @@ The complete import process is as follows:
 
 2. `tidb-lightning` creates the skeleton of all tables from the data source.
 
-3. Each table is split into multiple continuous *batches*, so that data from a huge table (200 GB+) can be delivered incrementally.
+3. Each table is split into multiple continuous *batches*, so that data from a huge table (200 GB+) can be delivered incrementally and concurrently.
```

Comment:
```suggestion
3. Each table is split into multiple continuous *batches*, so that data from a huge table (200 GB+) can be imported incrementally and concurrently.
```